### ISSUE TITLE
Adds JS$ prefix for set() and get() methods

### ISSUE THREAD

**Issue by pulyaevskiy** [dart-lang/js_facade_gen#31]
In Firestore.d.ts:

```ts
export class DocumentReference {
  set(data: DocumentData, options?: SetOptions): Promise<WriteResult>;
  get(): Promise<DocumentSnapshot>;
}
```

Result:

```dart
@JS("FirebaseFirestore.DocumentReference")
class DocumentReference {
  external Promise<WriteResult> JS$set(DocumentData data, [SetOptions options]);
  external Promise<DocumentSnapshot> JS$get();
}
```

Curious what's the reason for this prefix? I tried removing `JS$` prefix and everything still works.

**Comment by thosakwe** [comment]
AFAIK methods in Dart can't be named `get` or `set`

**Comment by jacob314** [comment]
those renames were because I thought dart methods can't be named get or set. If that isn't the case, the facade generator tool can be tweaked.

**Comment by johnpryan** [comment]
methods can be named `get` and `set` - the polymer_interop package defines get and set methods.  `package:http` is another example (Client.get())

**Comment by derekxu16** [comment]
Fixed by #64
